src/clarity-angular/layout/nav/_responsive-nav.clarity.scss

```diff
@@ -189,7 +190,8 @@ $clr-transition-style: ease;
             .header-nav.clr-nav-level-1,
             .subnav.clr-nav-level-1,
             .sub-nav.clr-nav-level-1,
-            .sidenav.clr-nav-level-1 {
+            .sidenav.clr-nav-level-1,
+            .clr-vertical-nav.clr-nav-level-1 {
```

Comment:
yep. i have noted this down already. am not changing it in this PR though because this is part of the CSS cleanup since this was written before vertical nav. thanks for raising this though. i hate this and the header css a lot.